tests/tests_integration_pytest.yml

```diff
@@ -0,0 +1,66 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- name: Empty play to gather facts
+  hosts: all
+
+- name: Run the role once first
+  hosts: all
+  roles:
+    - linux-system-roles.network
+
+- import_playbook: playbooks/centos7_deps.yml
+  when:
+    - ansible_distribution == "CentOS"
+    - ansible_distribution_major_version >= '7'
+
+- import_playbook: playbooks/fedora32_deps.yml
```

Comment:
Where did CentOS 8 go? I remember that it worked there, already, didn't it?

Reply:
I just thought about using the same playbook for both, so I changed this line 14 to `>= '7'`

Reply:
In general this would be possible but currently it is only imported on Fedora 32, so it would not be used on CentOS 8 as far as I can see. Once we have something working for the distributions, the next step would be to clean it up using the variables as suggested in the other comment and group these packages maybe by other features:

- if it is CentOS/RHEL: set the pre_pacakges to epel-release
- if it is python3: Use python3-pip
- all: install git, rsync
- all: install pytest via pip